docs/index.md

```diff
@@ -142,6 +143,47 @@ Then run tests with:
 $ npm test
 ```
 
+## Run Cycle Overview
+
+A brief outline on the order Mocha's components are executed.
+Worth noting that all hooks, `describe` and `it` callbacks are run in the order they are defined (i.e. found in the file).
```

Comment:
Add comma after "id est".

Reply:
Sorry I don't follow where you mean here.

Reply:
Would have preferred to just reference [FIFO](https://en.wikipedia.org/wiki/FIFO_(computing_and_electronics)), but unsure if that would be too "jargony".

Reply:
Sorry i'm fairly sure `i.e.,` is not grammatically correct

Reply:
Most online sources I found say to add it.
* [OntarioTraining](https://ontariotraining.net/grammar-tip-punctuation-with-i-e-and-e-g/)
* [Grammarly](https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/)
* [ProofreadNow](https://www.proofreadnow.com/blog/ie-vs-eg-and-when-to-use-which)
* [GrammarGirl](https://www.quickanddirtytips.com/education/grammar/ie-versus-eg?page=1)
* [StackExchange](https://english.stackexchange.com/questions/16172/should-i-always-use-a-comma-after-e-g-or-i-e)

GrammarGirl's research showed 5 of 6 style guides recommend the comma, but I note (in the last link) that British form usually omits it.

Don't know if Mocha has declared a favored language flavor for its documentation.

Reply:
@plroebuck do you feel strongly about this? 
I felt if everywhere accepts `i.e.` but only certain areas accept `i.e.,` then it was more fair to opt for the option more globally supported.
If this is blocking approval/merge I will change it.

Reply:
Preferred, but not forcing you to - this is not the blocking issue.